docs/admin/upgrade/upgrade-installation.md

```diff
@@ -4,6 +4,7 @@ weight: 21
 type: "docs"
 aliases:
   - /docs/install/upgrade-installation
+  - /docs/upgrade/upgrade-installation
```

Comment:
Maybe rename this to `upgrade/yaml.md`, and the other to `upgrade/operator.md`?

Reply:
I'm also okay if you don't want to have to make a lot of superfluous internal link updates in this PR.

Reply:
Yeah, I'd rather not add too many :) I'll add it as a todo comment maybe?

Reply:
It actually looks like it's not linked to from anywhere so I updated them to upgrade-yaml and upgrade-operator just to make it clearer